ss58-registry.json

```diff
@@ -477,6 +477,15 @@
 			"decimals": [12],
 			"standardAccount": "*25519",
 			"website": "https://crust.network"
+		},
+		{
+			"prefix": 67,
+			"network": "equilibrium",
+			"displayName": "Equilibrium Network",
+			"symbols": ["Unknown", "USD", "EQ", "ETH", "BTC", "EOS", "DOT", "CRV"],
+			"decimals": [0,9,9,9,9,9,9,9],
```

Comment:
I think this should only represent assets tracked by instances of the Balances pallet. The latest Assets pallet stores metadata like symbol/decimals for registered assets on chain. Are all of these native tokens on Equilibrium?

Reply:
We have our version of Balances pallet with this tokens. As far as I understand symbols and decimals are needed for polkadot.js UI to display the correct account balances in all tokens.

Reply:
Native token is only EQ

Reply:
But all of this tokens are in one pallet instance

Reply:
We use a DoubleMap<Address, Currency> storage for token balances instead of Map<Address>, so the idea is to bind the Currency param in custom balance getter function https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/balances/account.ts#L66

Reply:
So do you really want to register for 8 native currencies?

Reply:
We have this tokens in our chain and we want to proper display them in accounts tab. Native token is EQ, but we are gonna to bridge all of other tokens.